.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  pull_request: {}
+  push:
+    branches:
+      - master
```

Comment:
I think it should be great to add the `workflow_dispatch` trigger here. It will be helpful if someone wants to run these checks with their own branch.

Reply:
Makes sense to me. Added in 3e6a93f.